spring-batch-samples/src/test/java/org/springframework/batch/sample/AMQPJobFunctionalTests.java

```diff
@@ -34,7 +35,7 @@
  * queue for inspection.</p>
 */
 
-@RunWith(SpringJUnit4ClassRunner.class)
+@ExtendWith(SpringExtension.class)
 @ContextConfiguration(locations = { "/simple-job-launcher-context.xml", "/jobs/amqp-example-job.xml", "/job-runner-context.xml" })
```

Comment:
Not really important, but if `@ContextConfiguration` is replaced with `@SpringJUnitConfig`, the `@ExtendWith(SpringExtension.class)` can be omitted, which makes the code a bit leaner.

`@SpringJUnitConfig` is basically `@ConfigurationConfiguration` but meta-annotated with `@ExtendWith(SpringExtension.class)`.

Reply:
+1 to using `@SpringJUnitConfig` here and in every test class changed in this PR. Another PR could be opened to update all tests cases to use this annotation where appropriate.

Reply:
The tests have been migrated to JUnit 5 and `@SpringJUnitConfig` in #4173. When the PR is rebased, the tests in the sample module shouldn't require much change.